### PR TITLE
Feature: Send Environment container logs to cloudwatch using fluentd

### PR DESCRIPTION
**Pull request by Ayukha**
- [x] created namespace
- [x] created service account
- [x] created cluster_role
- [x] created cluster_role_binding
- [x] created config_map
- [x] installed fluentd

### CODE REVIEW

File: scripts/workers/rl_submission_worker.py [hunk 1]
```diff
@@ -274,6 +287,555 @@ def install_gpu_drivers(api_instance):
...
+            raise
+
+
+def create_config_map(api_instance):
+    """Function to create a config_map on AWS EKS cluster
+    Arguments:
+        api_instance {[AWS EKS API object]} -- API object for creating config_map
+    """
+    namespace = "amazon-cloudwatch"
+    fluent_conf = r"""
```
Comment (deshraj) [inline scripts/workers/rl_submission_worker.py:392]: Can you please put the conf in a file?

**Comment by RishabhJain2018** [comment]
@Ayukha Any updates here?

File: scripts/workers/code_upload_submission_worker.py [hunk 2]
```diff
@@ -274,6 +288,241 @@ def install_gpu_drivers(api_instance):
...
+            raise
+
+
+def install_fluentd(api_instance):
+    """Function to get the status of a running job on AWS EKS cluster
+    Arguments:
+        api_instance {[AWS EKS API object]} -- API object for creating deamonset
+    """
+    logging.info("Installing Fluentd ...")
+    fluentd_manifest = """
```
Comment (RishabhJain2018) [inline scripts/workers/code_upload_submission_worker.py:423]: Can we please have it in fluentd_conf file.

File: scripts/workers/code_upload_submission_worker.py [hunk 3]
```diff
@@ -274,6 +288,241 @@ def install_gpu_drivers(api_instance):
...
+            kind="ServiceAccount",
+            name="fluentd",
+            namespace="amazon-cloudwatch",
+        ),
+    )
+    try:
+        api_instance.create_cluster_role_binding(body)
+    except ApiException as e:
+        if e.status == 409:
+            logging.info("cluster_role_binding Already created")
```
Comment (RishabhJain2018) [inline scripts/workers/code_upload_submission_worker.py:382]: Can you please be a bit more verbose here as which cluster is this and what binding is this?

File: scripts/workers/code_upload_submission_worker.py [hunk 4]
```diff
@@ -274,6 +288,241 @@ def install_gpu_drivers(api_instance):
...
+            api_groups=[""],
+            resources=["namespaces", "pods", "pods/logs"],
+            verbs=["get", "list", "watch"],
+        ),
+    )
+    try:
+        api_instance.create_cluster_role(body=body)
+    except ApiException as e:
+        if e.status == 409:
+            logging.info("cluster_role Already created")
```
Comment (RishabhJain2018) [inline scripts/workers/code_upload_submission_worker.py:353]: Can you please be a bit more verbose here as which cluster is this and what binding is this?

File: scripts/workers/code_upload_submission_worker.py [hunk 5]
```diff
@@ -274,6 +288,241 @@ def install_gpu_drivers(api_instance):
...
+        kind="ServiceAccount",
+        metadata=client.V1ObjectMeta(name="fluentd"),
+    )
+    try:
+        api_instance.create_namespaced_service_account(
+            namespace=namespace, body=body,
+        )
+    except ApiException as e:
+        if e.status == 409:
+            logging.info("Service Account Already created")
```
Comment (RishabhJain2018) [inline scripts/workers/code_upload_submission_worker.py:329]: Can you please be a bit more verbose here as which cluster is this and what binding is this?

File: scripts/workers/code_upload_submission_worker.py [hunk 6]
```diff
@@ -274,6 +288,241 @@ def install_gpu_drivers(api_instance):
...
+        kind="Namespace",
+        metadata=client.V1ObjectMeta(
+            name="amazon-cloudwatch", labels={"name": "amazon-cloudwatch"}
+        ),
+    )
+    try:
+        api_instance.create_namespace(body)
+    except ApiException as e:
+        if e.status == 409:
+            logging.info("Namespace Already created")
```
Comment (RishabhJain2018) [inline scripts/workers/code_upload_submission_worker.py:307]: Can you please be a bit more verbose here as which cluster is this and what binding is this?

**Comment by RishabhJain2018** [comment]
@Ayukha ?

**Comment by Ayukha** [comment]
Already added!

**Pull request closed without merging**